modules/ROOT/pages/reference/procedures.adoc

```diff
@@ -318,6 +318,16 @@ In 4.2, signature changed to `db.listLocks() :: (mode :: STRING, resourceType ::
 | label:yes[]
 | label:new[Introduced in 5.23] label:server-management[`SERVER MANAGEMENT` privilege only]
 
+| xref:reference/procedures.adoc#procedure_dbms_cluster_recreateDatabase[`dbms.cluster.recreateDatabase()`]
+| label:no[]
+| label:yes[]
+| label:new[Introduced in 5.24]
```

Comment:
```suggestion
| label:new[Introduced in 5.24] label:database-management[`DATABASE MANAGEMENT` privilege only]
```

Reply:
So in the docs [here](https://neo4j.com/docs/operations-manual/current/authentication-authorization/dbms-administration/#access-control-dbms-administration-database-management), at Table 23, I think we need the first two columns. But do you mean this is something which is usually written in text in the documenatation part instead?

Reply:
In a discussion elsewhere: The privileges will only be in the recreate doc under "Managing databases in a cluster" and "Recreate databases"